src/containers/ToDoList/ToDoList.jsx

```diff
@@ -31,8 +33,30 @@ export const ToDoList = () => {
     setTasks(tasks.filter((task) => task.id !== id));
   };
 
+  const handleAddNewTask = () => {
+    if (newTask.trim() !== "") {
```

Comment:
This condition is good, but what we always tend to do, in order to get rid of robust conditions is doing this:

```
if (newTask.trim() === "") {
  return;
}
```